Frontend/rasandbox/shared/axios/SteamRepositoryAxios.ts

```diff
@@ -11,6 +11,25 @@ export class SteamRepositoryAxios
   public loadGames() {
     return games;
   }
+  public filterGames(
+    genreFilter: string,
```

Comment:
Einigt euch hier bitte auf ein einheitliches interface von `Filter` was sowohl in Frontend als auch Backend verwendet wird.
Und lager dieses Interface bitte in eine separate .ts Datei aus, wo alle deine Interfaces liegen.

Reply:
DONE